.travis.yml

```diff
@@ -16,8 +16,7 @@ cache:
   - $HOME/.sbt/boot/
 
 script:
-  - sbt ++$TRAVIS_SCALA_VERSION clean multi-jvm:test
-  - sbt ++$TRAVIS_SCALA_VERSION coverage test integration:test
+  - sbt ++$TRAVIS_SCALA_VERSION clean coverage test integration:test
```

Comment:
Removed the `multi-jvm:test` because `test` also runs multi-jvm tests now (from the uncommented lines in `project/ClusterTests.scala`)